tensorflow/contrib/avro/BUILD

```diff
@@ -0,0 +1,126 @@
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+load("//tensorflow:tensorflow.bzl", "tf_custom_op_py_library")
+load("//tensorflow:tensorflow.bzl", "tf_custom_op_library")
+load("//tensorflow:tensorflow.bzl", "cc_header_only_library")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_wrapper_py")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_libs")
+
+tf_custom_op_py_library(
+    name = "avro",
+    srcs = glob(["python/*.py"]),
+    dso = [":python/_avro_record_dataset.so",
+           ":python/_parse_avro_record.so",
+           ":python/libavro.so"
+    ],
+    kernels = [
+        ":avro_record_dataset_op_lib",
+        ":parse_avro_record_op_lib",
+    ],
+    deps = [
+         ":avro_record_dataset",
+         ":parse_avro_record",
+    ],
+    srcs_version = "PY2",
+    visibility = ["//visibility:public"],
+)
+
+# Tensorflow will link to libavro.so correctly, but it will leave it an external bazel
+# directory, which won't exist in the generated pip package. Therefore, we copy libavro.so
+# into the same directory -- of the pip package.
+# TODO(fraudies): Remove this copy rule;
+genrule(
```

Comment:
Could we assume that libavro.so is on the library path?

Reply:
One way that works is how boosted trees does it:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/BUILD#L69

Reply:
I'm removing it for now.